baseplate/experiments/providers/feature_flag.py

```diff
@@ -7,7 +7,9 @@
 
 
 class FeatureFlag(R2Experiment):
-    """An experiment with a single variant "active".
+    """An experiment with a single variant "active". This type of experiment
```

Comment:
rather than overembiggening the summary line of the docstring, you can add a sphinx deprecation notice to the docs instead.

http://www.sphinx-doc.org/en/stable/markup/para.html#directive-deprecated

Reply:
fixed